### DISCUSSION TITLE
HPEOneView-6.30 -- Get-OVRemoteSupportEntitlementStatus

### DISCUSSION THREAD

**Discussion by rwlib3** [HewlettPackard/POSH-HPOneView#627] [Q&A]
Trying to determine how this cmdlet connects to HPE for a warranty check? Running from my local workstation against a remote OneView (Synergy chassis for example) results in the same thing. 

```powershell
Connect-OVMgmt -hostname $oneview -Credential $ovcred

Get-OVServer | Get-OVRemoteSupportEntitlementStatus

ResourceName ResourceSerialNumber IsEntitled EntitlementPackage EntitlementStatus CoverageDays OfferEndDate        
------------ -------------------- ---------- ------------------ ----------------- ------------ ------------        
             xxxxxxxxx           False                         INVALID                        1/1/0001 12:00:00 AM
```
             
I'm not seeing any firewall blocks from the OV instance.

Thanks for any insight into this!

**Comment by ChrisLynchHPE** [comment]
The device isn't being reported from the HPE PointNext backend with a valid warranty.  Does the UI show similar behavior?

**Reply by rwlib3** [reply]
Here's an actual example:
```powershell
Get-OVServer | Get-OVRemoteSupportEntitlementStatus
ResourceName ResourceSerialNumber IsEntitled EntitlementPackage EntitlementStatus CoverageDays OfferEndDate        
------------ -------------------- ---------- ------------------ ----------------- ------------ ------------        
             MXQ2280TVN           False                         INVALID                        1/1/0001 12:00:00 AM
             
```
HPE support site shows this serial as valid.
![fw7eFlbmo0](https://user-images.githubusercontent.com/37152147/210442982-34c31e76-bf26-4341-9d26-41295fb40893.png)

**Reply by ChrisLynchHPE** [reply]
I need the screenshot from the appliance, not the HPE Support Center portal.  Please go to Server Hardware, select the specific server in the inventory, then choose Remote Support from the submenu navigation.
